### PR TITLE
Add Create account page

### PR DESCRIPTION
**Pull request by yunpeng1234**
Address #9 

image

Basic validation and UI components are done. 

1. Alert box will pop when submitting different passwords
2. Successful validation resets form

Would suggest form validation libraries like Formik in the future

### CODE REVIEW

**Comment by kevinchua6** [comment]
1. Please help to create an issue to add form validation libraries such as Formik
2. Also need to set a minimum length for the password, possibly using Formik (create an issue)
3. The alert when passwords are different is also inconsistent: is there a way to make the behavior like email (with a dropdown message)?
image


Other than that, LGTM!

**Comment by yunpeng1234** [comment]
Issue #18 is added
Now the behaviour is defaulted to chakra UI form behaviour. After adding Formik it should be ok

**Comment by MorningLit** [comment]
Can you try incorporating React Hook Forms, removing usestate from forms and also remove the window alert message?https://chakra-ui.com/getting-started/with-hook-form  Other than that it should be lgtm 😃

**Comment by yunpeng1234** [comment]
#18 
Incorporated React hook form and changed behavior of form to just indicate if field is valid. CI/CD fail due to Post.id

**Review by MorningLit — APPROVED** [review]

File: pages/users/create.tsx [hunk 1]
```diff
@@ -0,0 +1,113 @@
...
+    <Flex width='full' align='center' justifyContent='center'>
+      <Box p={8} width='50%' maxWidth='1000px' borderWidth={1} borderRadius={8} boxShadow='md'>
+        <>
+          <Box textAlign='center'>
+            <Heading>Create Account</Heading>
+          </Box>
+          <Box textAlign='left'>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <FormControl isRequired isInvalid={Boolean(errors.email)}>
+                <FormLabel>Email</FormLabel>
```
Comment (MorningLit) [inline pages/users/create.tsx:43]: remember to add htmlFor="email" here

File: pages/users/create.tsx [hunk 2]
```diff
@@ -0,0 +1,113 @@
...
+  return (
+    <Flex width='full' align='center' justifyContent='center'>
+      <Box p={8} width='50%' maxWidth='1000px' borderWidth={1} borderRadius={8} boxShadow='md'>
+        <>
+          <Box textAlign='center'>
+            <Heading>Create Account</Heading>
+          </Box>
+          <Box textAlign='left'>
+            <form onSubmit={handleSubmit(onSubmit)}>
+              <FormControl isRequired isInvalid={Boolean(errors.email)}>
```
Comment (MorningLit) [inline pages/users/create.tsx:42]: remove isRequired

Comment (MorningLit) [reply pages/users/create.tsx:42]: and also for other FormControls

File: pages/users/create.tsx [hunk 3]
```diff
@@ -0,0 +1,113 @@
...
+              <FormControl isRequired mt={6} isInvalid={Boolean(errors.password)}>
+                <FormLabel>Password</FormLabel>
+                <InputGroup>
+                  <Input
+                    id='password'
+                    type={showPassword ? 'text' : 'password'}
+                    placeholder='*******'
+                    size='md'
+                    {...register('password', {
+                      required: 'Email is required',
```
Comment (MorningLit) [inline pages/users/create.tsx:67]: Update required error message

File: pages/users/create.tsx (hunk 1 above)
Comment (MorningLit) [reply pages/users/create.tsx:43]: and also for other labels

File: pages/users/create.tsx (hunk 3 above)
Comment (MorningLit) [reply pages/users/create.tsx:67]: and also for ConfirmPassword

**Review by MorningLit — CHANGES_REQUESTED** [review]

File: pages/users/create.tsx (hunk 1 above)
Comment (MorningLit) [reply pages/users/create.tsx:43]: https://developer.mozilla.org/en-US/docs/Web/HTML/Element/label reason for the "for" attribute, i've also tried wrapping the label over the input but there was a small margin error

**Comment by yunpeng1234** [comment]
@MorningLit Fixed the changes requested 🙏 
@s7u4rt99 Create page as changed so don't override when merging recoil PR🙏

**Review by kevinchua6 — APPROVED** [review]
LGTM

**Pull request merged**